### DISCUSSION TITLE
Any plan to add markers plugin in v7 in the future?

### DISCUSSION THREAD

**Discussion by diegoanazco79** [katspaugh/wavesurfer.js#2927] [Q&A]
In the old version 6.5.x there was a bookmarks plugin, and it added a lot of value to the project.

I would like to know if you plan to add this type of plugin again

**Comment by katspaugh** [comment]
This functionality is now covered by the Regions plugin.
A region with just a start time and no end time is a maker.
See this example: https://wavesurfer-js.org/examples/#regions.js

```
  regions.addRegion({
    start: 19,
    content: 'Hello',
    color: 'green'
  })
```

<img width="357" alt="Screenshot 2023-06-23 at 07 15 41" src="https://github.com/katspaugh/wavesurfer.js/assets/381895/047a9266-2564-42e6-bd7e-26cfbd9f84a2">

**Reply by eshy** [reply]
Is there away to have a custom element with the region plugin? I was using the markerElement to have a nice UI (see image) and can't figure out how to do it with the regions plugin
![image](https://github.com/katspaugh/wavesurfer.js/assets/322731/50af6725-9afd-4dea-8f01-dd24c6083687)

**Reply by katspaugh** [reply]
You can pass a custom DOM element in the `content` option when adding a region.

Or use CSS and the `::part(region)` selector. See https://wavesurfer-js.org/examples/#styling.js
